src/com/facebook/buck/jvm/java/intellij/ij-module.st

```diff
@@ -49,7 +49,14 @@
         %if(generatedFolder.isAndroidResources)% type="java-resource" %endif%/>
     </content>
 }%
+
+%if(!androidFacet.enabled && java_sdk)%
+    <orderEntry type="jdk" jdkName="%java_sdk%" jdkType="JavaSDK" />
```

Comment:
> I had a bunch of code that would attempt to read IntelliJ's config

What if there are multiple instalaltions of inteeli/android studio or its variants? There is a lot of complexity involved in that calculation and it should be kept separate from this change